backoff.py

```diff
@@ -218,10 +218,22 @@ def constant(interval=1):
         yield interval
 
 
+def random_jitter(value):
```

Comment:
I'll want to add docstrings to all these jitter functions before a release. I don't mind doing this later though.